e2e_tests/tests/test_system.py

```diff
@@ -471,3 +471,13 @@ def test_pytorch_parallel() -> None:
     exp.run_basic_test_with_temp_config(
         config, conf.official_examples_path("trial/mnist_pytorch"), 1
     )
+
+
+@pytest.mark.e2e_cpu  # type: ignore
+def test_fail_on_first_validation() -> None:
+    error_log = "failed on first validation"
```

Comment:
not sure what this variable is actually for?

Reply:
i forgot to change it, it's what is raised in the model def.